packages/jest-jasmine2/src/index.js

```diff
@@ -157,9 +164,14 @@ async function jasmine2(
 
   runtime.requireModule(testPath);
   await env.execute();
```

Comment:
shouldn't `env.execute()` be in try/catch too?

Reply:
Good point, lemme try that